renderers/browser-render.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
```

Comment:
Yup, in the client.js entry point: https://github.com/meetup/mup-web/blob/master/src/client.jsx#L1